Makefile

```diff
@@ -33,108 +33,8 @@ endif
 GO=go
 GOFLAGS=""
 
-# requires:
-#   AWS_SECRET_ACCESS_KEY       - Secret from AWS
-#   AWS_ACCESS_KEY_ID           - Access key from AWS
-#   S3_REGION                   - Region of the bucket used to store the model
-#   S3_ENDPOINT                 - Endpint of the bucket
-#   IMAGE_REGISTRY_USERNAME     - quay.io username
-#   IMAGE_REGISTRY_PASSWORD     - quay.io password
-#   GIT_SELF_SIGNED_CERT        - Self-signed cert to be used in pipeline by git (optional)
-#   S3_SELF_SIGNED_CERT         - Self-signed cert to be used in pipeline by the script that's used to download the model from S3-compatible object storage (optional)
-#   GIT_TOKEN                   - Auth token used by the git ops pipeline to make a pull request
-#   GIT_USERNAME                - Username linked to the GIT_TOKEN
-go-test-setup:
```

Comment:
You can remove this target from the `.PHONY` line

Reply:
👍